SampleTest/MainPage.xaml

```diff
@@ -2,16 +2,19 @@
     x:Class="SampleTest.MainPage"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:local="using:SampleTest"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:local="using:SampleTest"
     xmlns:wgt="using:Microsoft.Toolkit.Graph.Controls"
     xmlns:graph="using:Microsoft.Graph"
+    xmlns:global="using:System.Globalization"
     xmlns:Interactivity="using:Microsoft.Xaml.Interactivity"
     xmlns:providers="using:Microsoft.Toolkit.Graph.Providers"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    xmlns:ex="using:Microsoft.Toolkit.Extensions"
     mc:Ignorable="d"
     Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
-    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"><!-- Workaround for https://github.com/microsoft/microsoft-ui-xaml/issues/1286 -->
+    xmlns:controlsKeepThis="using:Microsoft.Toolkit.Uwp.UI.Controls"><!-- Workaround for https://github.com/microsoft/microsoft-ui-xaml/issues/1286 -->
```

Comment:
If we bump the target SDK, should this issue go away? If so, we should do this.

Reply:
Also, this is being added right here on the top(line 13), so I don't think you need it anymore.

Reply:
I might bump/test that as a separate PR?

Reply:
Added to #57